htmresearch/frameworks/union_temporal_pooling/activation/excite_functions/excite_functions_all.py

```diff
@@ -62,7 +62,7 @@ def excite(self, currentActivation, inputs):
     @param inputs            (numpy array) inputs for each cell
     """
 
-    currentActivation = self._minValue + (self._maxValue - self._minValue) / \
+    currentActivation += self._minValue + (self._maxValue - self._minValue) / \
```

Comment:
use implicit continuation with parenthesis rather than backslashes. If the equation gets too complex to follow, split it up into several calculations.